.gitignore

```diff
@@ -8,4 +9,4 @@ test/.project
 *.lock
 *.bak
 .idea/
-*.mws
```

Comment:
Missing end of line. Please configure your IDE to do add this.

Reply:
Added